Vagrantfile

```diff
@@ -45,7 +67,7 @@ Vagrant.configure(2) do |config|
   # the path on the host to the actual folder. The second argument is
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
-  config.vm.synced_folder ".", "/home/vagrant/hoot"#, group: "tomcat6"#, mount_options: ["dmode=775,fmode=775"]
+  config.vm.synced_folder ".", "/home/vagrant/hoot"#, group: "tomcat6", mount_options: ["dmode=775,fmode=775"]
```

Comment:
I forgot that the group option doesn't work when mounting shared folder via nfs.  We should add the # back in above so when virtulbox users uncomment the group option they don't also get the mount_options (needed on parallels provider on mac)